catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -199,7 +199,7 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
 					intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_SINGLE_TOP);
 					startActivity(intent);
 				} else {
-					ToastUtil.showError(getApplicationContext(), R.string.error_no_whatsapp);
+					ToastUtil.showError(getBaseContext(), R.string.error_no_whatsapp);
```

Comment:
I can't approve these changes. For the ToastUtil, I guess it doesn't matter which context you pass in?…

@thmq, since you're very experienced with Android, maybe you can take a look at the context changes?

Reply:
i used `getBaseContext()` instead of `getApplicationContext()` to reduce using the _**static_Contetxt**_